sycl/source/detail/device_info.hpp

```diff
@@ -283,16 +283,9 @@ struct get_device_info_impl<std::vector<memory_scope>,
 template <>
 struct get_device_info_impl<bool, info::device::ext_oneapi_bfloat16> {
   static bool get(RT::PiDevice dev, const plugin &Plugin) {
-
-    bool result = false;
-
-    RT::PiResult Err = Plugin.call_nocheck<PiApiKind::piDeviceGetInfo>(
-        dev, PiInfoCode<info::device::ext_oneapi_bfloat16>::value,
-        sizeof(result), &result, nullptr);
-    if (Err != PI_SUCCESS) {
-      return false;
-    }
-    return result;
+    // This aspect indicates support for bfloat16 math functions,
+    // which we don't do yet.
+    return false;
```

Comment:
Shouldn't this call into the PI layer?  I assume this is implementing the `aspect::ext_oneapi_bfloat16` aspect?

Reply:
Yes, this will go back to the way it was, with an actual query into the plugin to tell if bfloat16 math is supported.